Backend/app.py

```
@@ -174,15 +188,23 @@ def cleanup_cache():
     except Exception as e:
         return jsonify({"error": str(e)}), 500
 
+
 @app.route('/')
 def index():
-    from routes.appointments import get_appointments
-    from routes.tasks import get_tasks
-    appointment_db =  get_appointments()
+    appointment_db = get_appointments()
     task_db = get_tasks()
```

Ruff (0.14.10)

Local variable `task_db` is assigned to but never used

Remove assignment to unused variable `task_db`

(F841)